WordPress/WordPressTest/BlogVideoLimitsTests.swift

```diff
@@ -1,56 +0,0 @@
-import XCTest
-
-class BlogVideoLimitsTests: CoreDataTestCase {
-
-    private var blog: Blog!
-
-    override func setUpWithError() throws {
-        blog = NSEntityDescription.insertNewObject(forEntityName: "Blog", into: mainContext) as? Blog
-        blog.url = Constants.blogURL
-        blog.xmlrpc = Constants.blogURL
-    }
-
-    override func tearDownWithError() throws {
-        blog = nil
-    }
-
-    func testCanUploadAssetPaidPlan() throws {
-        // Given a blog
-        // When blog has a paid plan
-        blog.hasPaidPlan = true
-
-        // Then it can upload assets irrespective of allowance
-        XCTAssertTrue(blog.canUploadAsset(true))
-        XCTAssertTrue(blog.canUploadAsset(false))
-    }
-
-    func testCanUploadAssetWPCom() throws {
-        // Given a blog
-        // When blog is on WPCom and not paid
-        blog.isHostedAtWPcom = true
-        blog.hasPaidPlan = false
-
-        // Then it can upload assets when allowance not exceeded
-        XCTAssertTrue(blog.canUploadAsset(false))
```

Comment:
It was basically testing configuration, repeating the logic one-to-one – not very useful.
Since I removed the `blog.canUploadAsset` variant the test was using, I decided to remove the test as well.